Source/cmake/OptionsWin.cmake

```diff
@@ -142,6 +142,7 @@ endif ()
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_LAYOUT_TESTS PUBLIC ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_MATHML PUBLIC ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_MEDIA_CONTROLS_SCRIPT PUBLIC ON)
+WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_MANAGED_MEDIA_SOURCE PUBLIC OFF)
```

Comment:
No need for this: it's redundant with the default value in WebKitFeatures.cmake. (And not alphabetized.)

Reply:
isn't it redundant for ENABLE_MEDIA_SOURCE too ?

Reply:
Actually, it's not the same as the default value in WebKitFeatures.cmake which is based on the value of ENABLE_VIDEO ; here it's turned off.
But regardless all the code is dependent on ENABLE(MANAGED_MEDIA_SOURCE) && ENABLE(MEDIA_SOURCE) so if ENABLE_MEDIA_SOURCE is off as it is here, it will be disabled as well in the code.

Reply:
You have in WebKitFeatures.cmake:

` WEBKIT_OPTION_DEFINE(ENABLE_MANAGED_MEDIA_SOURCE "Toggle Managed Media Source support" PRIVATE OFF)`

So it's OFF by default. The dependencies will ensure the feature gets unset if required, but the dependencies won't ever enable the feature.